scratch-vm/rspack.config.js

```
@@ -0,0 +1,215 @@
+const { rspack } = require('@rspack/core');
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)